### PR TITLE
Add package rfc3339

### PR DESCRIPTION
**Pull request by Skrylar**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -15687,5 +15687,16 @@
     "description": "Library for finding the differences between two sequences",
     "license": "Apache-2.0",
     "web": "https://github.com/mark-summerfield/diff"
+  },
+  {
+    "name": "rfc3339",
+    "url": "https://git.sr.ht/~skrylar/skrfc3339",
+    "method": "git",
+    "tags": [
+      "rfc3339,date,datetime,time"
```
Comment (dom96) [inline packages.json]: Each tag should be its own item

**Comment by Skrylar** [comment]
That was all generated with nimble /shrug

**Comment by dom96** [comment]
Yes, Nimble should detect this. But it's instructions are clear: separate tags with spaces not commas.

**Comment by narimiran** [comment]
> `E: rfc3339: a package by that name already exists.`

...and it is your package hosted on Github: https://github.com/Skrylar/rfc3339

**Comment by Skrylar** [comment]
Was just trying to do a version bump. Doesn't look like this is necessary anyhow.

**Pull request closed without merging**